docs/guides/sre/tls/configuring-tls-acme-client.md

```diff
@@ -0,0 +1,122 @@
+---
+slug: /guides/sre/configuring-tls-acme-client
+sidebar_label: 'Configuring automatic TLS provisioning via ACME'
+sidebar_position: 20
+title: 'Configuring ACME client'
+description: 'This guide provides simple and minimal settings to configure ClickHouse to use OpenSSL certificates to validate connections.'
+keywords: ['ACME configuration', 'TLS setup', 'OpenSSL certificates', 'secure connections', 'SRE guide', 'Let`s Encrypt']
+doc_type: 'guide'
+---
+
+import SelfManaged from '@site/docs/_snippets/_self_managed_only_automated.md';
+import configuringSsl01 from '@site/static/images/guides/sre/configuring-ssl_01.png';
+import Image from '@theme/IdealImage';
+import ExperimentalBadge from '@theme/badges/ExperimentalBadge';
+
+# Configuring automatic TLS provisioning via ACME
+
+<ExperimentalBadge/>
+<SelfManaged />
+
+This guide provides describes how to configure ClickHouse to use ACME (RFC8555) protocol.
+With ACME support, ClickHouse can obtain and renew certificates from providers like Let's Encrypt or ZeroSSL automatically.
+
+## Overview {#overview}
+
+ACME protocol defines automatic certificate update process with services like Let's Encrypt or ZeroSSL. In short, certificate requester needs to confirm domain ownership via predefined challenge types in order to get a certificate.
+
+To enable ACME, configure HTTP and HTTPS ports along with the `acme` block:
+
+```yaml
+<http_port>80</http_port>
+<https_port>443</https_port>
+
+<acme>
+    <email>valid_email@example.com</email>
+    <terms_of_service_agreed>true</terms_of_service_agreed>
+    <domains>
+        <domain>example.com</domain>
+    </domains>
+</acme>
+```
+
+The HTTP port serves ACME `HTTP-01` challenge requests during domain validation. Once validation completes and a certificate is issued, the HTTPS port serves encrypted traffic using the obtained certificate.
```

Comment:
`HTTP-01 challenge` is a ACME-specific thing - people not familiar with ACME will not know what it is. Maybe link it to some explanation on Wikipedia or the Let's Encrypt homepage?